completion/changes.cwl

```diff
@@ -0,0 +1,108 @@
+# package changes
+# fkastner 2021/03/17 for changes v4.0.1
+# URL: https://ctan.org/tex-archive/macros/latex/contrib/changes
+
+#include:todonotes
+#include:truncate
+#include:ulem
+#include:xcolor
+#include:xifthen
+#include:xkeyval
+#include:xstring
+
+\definechangesauthor[keyvals]{authorid}#s#%authorid
+\definechangesauthor{authorid}#s#%authorid
+#keyvals:\definechangesauthor
+name=
+color=#%color
+#endkeyvals
+
+\added[keyvals]{text}
+\added{text}
+\deleted[keyvals]{text}
+\deleted{text}
+\replaced[keyvals]{new%text}{old%text}
+\replaced{new%text}{old%text}
+\highlight[keyvals]{text}
+\highlight{text}
+#keyvals:\added,\deleted,\replaced,\highlight
+id=#%authorid
+comment=
+#endkeyvals
+
+\comment[keyvals]{text%todo}
+\comment{text%todo}
+#keyvals:\comment
+id=#%authorid
+#endkeyvals
+
+\listofchanges
+\listofchanges[keyvals]
+#keyvals:\listofchanges
+style=#list,summary,compactsummary
+title=
+show=#all,added,deleted,replaced,highlight,comment
+#endkeyvals
+
+# autogenerated by txs
+\ChangesListline{arg1}{arg2}{arg3}{arg4}#S
+\changesaddname#S
+\changesanonymousname#S
+\changesauthorname#S
+\changescommentname#S
+\changesdeletename#S
+\changeshighlightname#S
+\changesnochanges#S
+\changesnoloc#S
+\changesnosoc#S
+\changesreplacename#S
+\cleaders#S
+\compactsummaryofchangesname#S
+\csname#S
+\ifIsEmpty{arg1}{arg2}{arg3}#S
+\listofchangesname#S
+\protect#S
+\setaddedmarkup{arg1}#S
+\setauthormarkupposition{arg1}#S
+\setauthormarkuptext{arg1}#S
+\setauthormarkup{arg1}#S
+\setcommentmarkup{arg1}#S
+\setdeletedmarkup{arg1}#S
+\sethighlightmarkup{arg1}#S
+\setsocextension{arg1}#S
+\setsummarytowidth{arg1}#S
+\setsummarywidth{arg1}#S
+\settruncatewidth{arg1}#S
+\summaryofchangesname#S
+\theauthorcommentcount
+\\added#S
```

Comment:
I would prefer if the useless stuff below is removed (artifacts from the scan process)